app/src/main/java/org/simple/clinic/home/overdue/search/OverdueSearchUiRenderer.kt

```diff
@@ -7,20 +7,11 @@ class OverdueSearchUiRenderer(
 ) : ViewRenderer<OverdueSearchModel> {
 
   override fun render(model: OverdueSearchModel) {
-    if (!model.hasSearchQuery) {
+    if (!model.hasSearchQuery && !model.hasOverdueSearchResults) {
```

Comment:
Now that we are no longer having search results in model. We can remove the `hasOverdueSearchResults` check? WDYT?

Reply:
We can also drop the code related to search results in the model

Reply:
We are using this to show/hide search history and overdue search result recycler view. If we don't have any search results and we don't have a search query then we show the search history for example.

Reply:
But since we no longer want to update the search results in the model. This wouldn't work no? For example, we can now just hide the search history when the search query is not empty and hide the search results when the search query is empty.

Reply:
Yeah but then what about the case when the search query is empty and we clicked on a search history item and search results are shown. In that case also we need to hide the search history list

Reply:
So, we can hide the search results list when search query is empty and show search history list instead, and vice versa.

Reply:
So I understand why the current logic is wrong. But, we still have to handle the case I talked about above. Hiding search history when search query is empty and search results are loaded